charts/k8s-monitoring/docs/examples/namespace-labels-and-annotations/values.yaml

```
@@ -0,0 +1,74 @@
+---
+cluster:
+  name: pod-labels-and-annotations
+
+destinations:
+  - name: prometheus
+    type: prometheus
+    url: http://prometheus.prometheus.svc:9090/api/v1/write
+  - name: loki
+    type: loki
+    url: http://loki.loki.svc:3100/api/push
+  - name: tempo
+    type: otlp
+    url: http://tempo.tempo.svc
+    metrics: {enabled: false}
+    logs: {enabled: false}
+    traces: {enabled: true}
+
+clusterMetrics:
+  enabled: true
+
+podLogs:
+  enabled: true
+
+
+
+applicationObservability:
```

Make yaml lint happy
```suggestion
podLogs:
  enabled: true

applicationObservability:
```